app/eventyay/plugins/exhibitor/__init__.py

```diff
@@ -0,0 +1,3 @@
+__version__ = '1.0.0'
+
+default_app_config = 'eventyay.plugins.exhibitor.apps.ExhibitorApp'
```

Comment:
<img width="1478" height="287" alt="image" src="https://github.com/user-attachments/assets/d44dcf4b-8d2f-4bfd-a7ca-2f8e3978dd77" />

Add a blank line at the end of file.